src/HomePage/AboutMe/AboutMe.scss

```diff
@@ -0,0 +1,26 @@
+.about-section {
+  display: flex;
```

Comment:
Flex is fine but it's a bit overkill for this use case as its just 2 block elements which stay ontop of each other anyway. Its good for centering vertically so i wont say definitely remove it but it does seem slightly unneccessary